packages/core/src/codewhisperer/models/model.ts

```diff
@@ -330,17 +330,19 @@ export class ZipManifest {
     hilCapabilities: string[] = ['HIL_1pDependency_VersionUpgrade']
     transformCapabilities: string[] = ['EXPLAINABILITY_V1'] // TO-DO: for SQL conversions, maybe make this = []
     customBuildCommand: string = 'clean test'
-    requestedConversions: {
-        sqlConversion:
-            | {
-                  source: string | undefined
-                  target: string | undefined
-                  schema: string | undefined
-                  host: string | undefined
-                  sctFileName: string | undefined
-              }
-            | undefined
-    } = { sqlConversion: undefined }
+    requestedConversions:
+        | {
+              sqlConversion:
+                  | {
+                        source: string | undefined
+                        target: string | undefined
+                        schema: string | undefined
+                        host: string | undefined
+                        sctFileName: string | undefined
+                    }
+                  | undefined
+          }
+        | undefined = undefined
```

Comment:
This would look nicer and also IMO be less confusing since we are removing so much text.
```suggestion
    requestedConversions?: {
        sqlConversion?: {
            source?: string
            target?: string
            schema?: string
            host?: string
            sctFileName?: string
        }
    }
```